scripts/prepare-release.sh

```diff
@@ -10,10 +10,11 @@ git checkout master
 git pull origin master
 git checkout develop
 git pull origin develop
+git checkout -b release-branch-"$(date +%s)"
```

Comment:
I think we should use `release/<date>` as the branch name. Use those namespaces!

Reply:
changed it to release/"$(date +%m-%d-%y.%S)"